reference/components/cf-serverd.markdown

```diff
@@ -344,44 +344,43 @@ The sequence of events is this:
 The full configuration would look something like this
 
 ```cf3
-        #########################################################
-        # Server config
-        #########################################################
+#########################################################
```

Comment:
This is the only section that made me pause. Which "full configuration" file would this be in? If you were modifying controls/cf_execd.cf then you probably wouldn't change to my_access_rules bundle name. If you were creating a replacement you would need to change promises.cf inputs[cf_execd].

Reply:
OK, I have tweaked it very slightly. This part of the documentation is about the configuration for collect_calls access. This "full configuration" snippet shows all of the various settings related to enabling client-initiated reporting. It's not specific to MPF. If you have any suggestions to clarify further please let me know. It's challenging to straddle the difference between documentation about core, and then conventions from MPF which won't necessarily apply if you run a completely custom policy set.

Reply:
Cool. I think for me I would like to see the text before the example say: change controls/cf_serverd.cf as follows (and then the two examples), or if my_access_rules() is really just augmenting what is in cf_server.cf, which it is, then you might break that section apart and prefix it with some text like: include this `bundle server my_access_rules()` in a separate file and include in inputs with augments (def.json). 

Something like that make any sense to you?

Reply:
So, the access rules are already present by default, and we have augments about them.

Again, this documetation about the control body is about the core component with has nothing really to do with MPF. I could however add some See Also references to the MPF augments for these settings.